app/templates/components/modals/change-image-modal.hbs

```diff
@@ -0,0 +1,50 @@
+<i class="black close icon"></i>
+<div class="header">
+  {{t 'Update image'}}
+</div>
+<br>
+<br>
+<div class="ui grid">
```

Comment:
Model content should be inside a div with class `content`. 
https://semantic-ui.com/modules/modal.html 

@sumedh123 I'm not sure why you haven't yet seen semantic UI docs properly.